frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -33,12 +33,22 @@ const HorizontalNonButtonContent = styled.div`
 `
 
 const StyledConfirmDialog = styled.div`
```

Comment:
We can probably rename this to something other than a dialog since it’s not being used that way anymore